admin_guide/scheduling/scheduler-advanced.adoc

```diff
@@ -0,0 +1,51 @@
+[[admin-guide-scheduling-advanced]]
+= Advanced Scheduling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Advanced scheduling involves configuring a pod so that the pod is required to run on particular nodes or has a preference to run on particular nodes.
+
+Generally, advanced scheduling  is not necessary, as the {product-title} automatically places pods in a reasonable manner. For example, the default scheduler attempts to distribute pods across the nodes evenly and considers the free resources in a node. However, you might want more control over where a pod is placed.
```

Comment:
A suggestion: s/free/available

I think "free" might come with "not having to pay money" connotations.